plugins/europeana/record.js

```diff
@@ -336,6 +346,41 @@ export default (axiosOverrides) => {
   };
 };
 
+const reduceLangMapsForLocale = (value, locale) => {
+  if (Array.isArray(value)) {
+    return value.map(val => reduceLangMapsForLocale(val, locale));
+  } else if (typeof value === 'object') {
+    if (isLangMap(value)) {
+      const selectedLocale = selectLocaleForLangMap(value, locale);
+      const langMap = {
+        [selectedLocale]: value[selectedLocale]
+      };
+      return Object.freeze(langMap);
+    } else {
+      return Object.keys(value).reduce((memo, key) => {
+        memo[key] = reduceLangMapsForLocale(value[key], locale);
+        return memo;
+      }, {});
+    }
+  } else {
+    return value;
+  }
+};
+
+const isLangMap = (value) => {
+  return (typeof value === 'object') && Object.keys(value).every(key => {
+    // TODO: is this good enough to determine lang map or not?
+    return key.length === 2 || key.length === 3;
+  });
+};
+
+const reduceEntity = (entity) => {
+  return Object.freeze({
+    about: entity.about,
+    prefLabel: entity.prefLabel
```

Comment:
Probably needs to include altLabel, maybe even hiddenLabel?

Reply:
Not at present.